api/src/system/getProvisioningState.ts

```diff
@@ -0,0 +1,50 @@
+import { HttpResponse } from "../httpd/lib";
```

Comment:
this is the wrong location for this kind of code.
This is the event sourcing of the system_information stream and should be located in the domain layer called system_information_eventsourcing. I would code it like the eventsourcing we already have.
a source function which applies all events where the apply method decides how to handle/apply the type of event.
Check out api/src/service/domain/workflow/global_permissions_eventsourcing.ts for an example

Reply:
system considered calls are set in `router.ts`. This is the same patter that is used for` /liveness ` `/readiness`  `/version`

Reply:
done